src/utils/moduleAssets.js

```diff
@@ -19,7 +20,12 @@ const getModuleAssetTitle = (t = str => str) => ({
 });
 
 
-const selectSchema = moduleAssetType => MODULE_ASSETS_MAP[moduleAssetType].schema;
+const retrieveSchemas = (network) => {
+  Object.values(MODULE_ASSETS_NAME_ID_MAP).forEach(async (moduleAssetId) => {
```

Comment:
I don't think we need to make several requests. If I remember correctly, you can make the request without any parameters and it should return all the schemas (which is fine for our case).